### ISSUE TITLE
Incorrect Peer Dependency Warning

### ISSUE THREAD

**Issue by aafaque33** [runette/ngx-leaflet-fullscreen#4]
Hi, 

While running the *npm install* or *yarn* receive the warning about wrong peer dependency. 

```
warning " > @runette/ngx-leaflet-fullscreen@1.0.4" has incorrect peer dependency "@angular/common@^8.2.14".
warning " > @runette/ngx-leaflet-fullscreen@1.0.4" has incorrect peer dependency "@angular/core@^8.2.14".
```

I checked the package.json in this project and that have

```
"peerDependencies": {
    "@angular/common": "^8.2.14",
    "@angular/core": "^8.2.14"
  }
```

Should this updated to Angular 12 ? As I have verified that it works with that version ...

**Comment by runette** [comment]
I will update that

**Comment by runette** [comment]
Fixed in Version 1.0.6 - now on npm. This version has been built using ng-packagr and Ivy partial compilation which means it should be compatible with all versions of Angular.io after version 12
